starter-core/src/main/java/io/micronaut/starter/feature/database/HibernateGorm.java

```diff
@@ -70,12 +70,8 @@ public void processSelectedFeatures(FeatureContext featureContext) {
     @Override
     public void apply(GeneratorContext generatorContext) {
         Map<String, Object> config = generatorContext.getConfiguration();
-        config.put("dataSource.url", "jdbc:h2:mem:devDb;MVCC=TRUE;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE");
         config.put("dataSource.pooled", true);
         config.put("dataSource.jmxExport", true);
-        config.put("dataSource.driverClassName", "org.h2.Driver");
-        config.put("dataSource.username", "sa");
-        config.put("dataSource.password", "");
```

Comment:
This will definitely cause GORM applications to not function. We can't assume because the H2 feature is added that it is using `micronaut-jdbc` and expecting `datasources.default`. I think you'll need to leave the configuration up to the individual feature, however perhaps add a method or methods on the driver feature to make that easier.

Reply:
Fixed

Reply:
This config is still being removed, so this will still cause GORM applications to not function

Reply:
fixed